modules/creating-serverless-apps-kn.adoc

```diff
@@ -16,12 +16,14 @@
 ----
 $ kn service create <SERVICE-NAME> --image <IMAGE> --env <KEY=VALUE>
 ----
-+
-.Example
-+
+
+.Example command
+----
+$ kn service create hello --image docker.io/openshift/hello-openshift --env TARGET=Knative
```

Comment:
Per https://github.com/openshift/origin/tree/master/examples/hello-openshift 

The openshift/hello-openshift doesn't use "TARGET" env, 

but it does have "RESPONSE",  so we could perhaps just set RESPONSE="Hello Knative"  or similar.